test/OrchardCore.Tests/Lucene/Context/SiteContext.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Security.Principal;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using Newtonsoft.Json;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Environment.Shell.Scope;
+using OrchardCore.Tenants.ViewModels;
+using OrchardCore.Tests.Apis.Context;
+
+namespace OrchardCore.Tests.Lucene
+{
+    public class SiteContext : IDisposable
+    {
+        public static OrchardTestFixture<SiteStartup> Site { get; }
```

Comment:
Cool!

But as you can see from the failing tests the `Site` fixture is static, so they fail, because they are trying to write to the same `App_Data` folder, as the Api tests, which is already in use, because it is running the Api tests.

I would suggest it might be easier if they remained as part of the Api fixture, perhaps even calling into the Lucene Api Controller to run them (but just an idea, not necesarily essential to me)